### ISSUE TITLE
Advanced Filtering for Calculation Groups

### ISSUE THREAD

**Issue by m-kovalsky** [otykier/TabularEditor#842]
**Describe the bug**
Attempting to filter the model to just calculation groups using the advanced filter. I've tried several methods which technically should work but don't.

**To Reproduce**
1. Paste any of the following into the advanced filter textbox.

:ObjectType="CalculationGroupTable"
:ObjectType = "CalculationGroup"
:ObjectTypeName="Calculation Group Table"
:ObjectTypeName="CalculationGroupTable"
:ObjectTypeName="CalculationGroup"
:SourceType="CalculationGroup"

**Expected behavior**
Is there another way to filter the model to just calculation groups?

**Observed behavior**
I did notice that the filter below works (and shows calc groups & calc items).

:ObjectType = "CalculationItem"

**Application specifics**
 - Tabular Editor version: 2.16.0

**Comment by otykier** [comment]
Fixed in [2.16.1](https://github.com/otykier/TabularEditor/releases/tag/2.16.1).

Can use the following syntaxes (CalculationGroupTables were not indexed prior to this fix, so there was no way to locate them using LINQ filtering):

```csharp
:ObjectType="CalculationGroupTable"
:ObjectTypeName="Calculation Group Table"
```

**Comment by m-kovalsky** [comment]
Thanks!
